diff --git a/R/Init.R b/R/Init.R
index c1301509..5768cd82 100644
--- a/R/Init.R
+++ b/R/Init.R
@@ -1,6 +1,6 @@
 #  MIT License
 #
-#  Copyright (c) 2017-2024 TileDB Inc.
+#  Copyright (c) 2017-2023 TileDB Inc.
 #
 #  Permission is hereby granted, free of charge, to any person obtaining a copy
 #  of this software and associated documentation files (the "Software"), to deal
@@ -55,15 +55,6 @@
 
     ## call setter for Rcpp plugin support
     .set_compile_link_options()
-
-    lib_path <- system.file("lib", .Platform$r_arch, paste0("libconnection", .Platform$dynlib.ext), package = "tiledb")
-    res <- dyn.load(lib_path)
-    .Call(`_tiledb_tldb_init_`, res$new_connection$address, PACKAGE="tiledb")
-}
-
-.onUnload <- function(libname) {
-    lib_path <- system.file("lib", .Platform$r_arch, paste0("libconnection", .Platform$dynlib.ext), package = "tiledb")
-    dyn.unload(lib_path)
 }
 
 .onAttach <- function(libname, pkgname) {
