diff --git a/src/Makevars.win b/src/Makevars.win
index 00fdfc7c..667d9fbd 100644
--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -16,18 +16,10 @@ PKG_LIBS = \
 	-lBCrypt -lKernel32 -lRpcrt4 -lWininet -lWinhttp -lWs2_32 -lShlwapi -lUserenv -lversion -lws2_32 \
 	-lsharpyuv
 
-LIB_CON = ../inst/lib$(R_ARCH)/libconnection.dll
-
-all: winlibs $(OBJECTS) $(SHLIB) $(LIB_CON)
-
-$(LIB_CON): connection/connection.o
-	mkdir -p $(dir $(LIB_CON))
-	$(SHLIB_LINK) $(LIBR) -o $@ $^
+all: clean winlibs
 
 winlibs:
 	"$(R_HOME)/bin$(R_ARCH_BIN)/Rscript.exe" "../tools/winlibs.R"
 
 clean:
-	rm -f $(SHLIB) $(OBJECTS) $(LIB_CON) connection/connection.o
-
-.PHONY: all clean
+	rm -f $(SHLIB) $(OBJECTS)
