diff --git a/inst/tinytest/test_vfs_file.R b/inst/tinytest/test_vfs_file.R
deleted file mode 100644
index 1993d49b..00000000
--- a/inst/tinytest/test_vfs_file.R
+++ /dev/null
@@ -1,13 +0,0 @@
-library(tinytest)
-
-tmp <- tempfile()
-ref <- as.raw(1:255)
-writeBin(ref, tiledb:::vfs_file(tmp))
-tst <- readBin(tiledb:::vfs_file(tmp),  raw(), 1000)
-expect_identical(tst, ref)
-
-tmp <- tempfile()
-ref <- as.character(mtcars)
-writeLines(ref, tiledb:::vfs_file(tmp))
-tst <- readLines(tiledb:::vfs_file(tmp))
-expect_identical(tst, ref)
