diff --git a/cleanup b/cleanup
index 2306aec0..ec6b596a 100755
--- a/cleanup
+++ b/cleanup
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-rm -f  src/Makevars src/*.o src/*.so config.log config.status inst/tiledb-*.tar.gz src/connection/*.o
-rm -rf tiledb.Rcheck autom4te.cache inst/tiledb/ inst/config.log inst/config.status tiledb/ inst/lib/
+rm -f  src/Makevars src/*.o src/*.so config.log config.status inst/tiledb-*.tar.gz
+rm -rf tiledb.Rcheck autom4te.cache inst/tiledb/ inst/config.log inst/config.status tiledb/
